lib/generators/clearance/routes/templates/routes.rb

```diff
@@ -1,12 +1,12 @@
 resources :passwords, controller: 'clearance/passwords', only: [:create, :new]
-  resource :session, controller: 'clearance/sessions', only: [:create]
+resource :session, controller: 'clearance/sessions', only: [:create]
 
-  resources :users, controller: 'clearance/users', only: [:create] do
-    resource :password,
-      controller: 'clearance/passwords',
-      only: [:create, :edit, :update]
-  end
+resources :users, controller: 'clearance/users', only: [:create] do
+  resource :password,
+    controller: 'clearance/passwords',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.